testing/tests/DevExpress.ui.widgets/menuBase.tests.js

```diff
@@ -333,6 +333,62 @@ QUnit.module('Menu rendering', () => {
 
         assert.equal($element.find('.dx-menu-item-content').first().text(), 'Test item');
     });
+
+    QUnit.test('Render item as link if url start with https', function(assert) {
+        const menuBase = createMenu({
+            items: [{ text: 'Item text', url: 'https://some_url' }]
+        });
+        const content = menuBase.element.find(`.${DX_MENU_ITEM_TEXT_CLASS}`).children()[0];
+
+        assert.strictEqual(content.tagName, 'A');
+        assert.strictEqual(content.getAttribute('href'), 'https://some_url');
+        assert.strictEqual(content.text, 'Item text');
+    });
+
+    QUnit.test('Render item as link if url start with http', function(assert) {
+        const menuBase = createMenu({
+            items: [{ text: 'Item text', url: 'http://some_url' }]
+        });
+        const content = menuBase.element.find(`.${DX_MENU_ITEM_TEXT_CLASS}`).children()[0];
+
+        assert.strictEqual(content.tagName, 'A');
+        assert.strictEqual(content.getAttribute('href'), 'http://some_url');
+        assert.strictEqual(content.text, 'Item text');
+    });
+
+    QUnit.test('Render item as simple text if url option is incorrect', function(assert) {
+        const menuBase = createMenu({
+            items: [{ text: 'Item text', url: '/some_url' }]
+        });
+        const content = menuBase.element.find(`.${DX_MENU_ITEM_TEXT_CLASS}`);
+
+        assert.ok(!content.children()[0]);
+        assert.strictEqual(content.text(), 'Item text');
+    });
+
+    QUnit.test('Provide extra attributes for link if linkAttr option set with url', function(assert) {
+        const menuBase = createMenu({
+            items: [{ text: 'Item text', url: 'http://some_url', linkAttr: { target: '_blank' } }]
+        });
+        const content = menuBase.element.find(`.${DX_MENU_ITEM_TEXT_CLASS}`).children()[0];
+
+        assert.strictEqual(content.tagName, 'A');
+        assert.strictEqual(content.getAttribute('href'), 'http://some_url');
+        assert.strictEqual(content.getAttribute('target'), '_blank');
+        assert.strictEqual(content.text, 'Item text');
+    });
+
+    QUnit.test('Should not raise error if linkAttr option is not object', function(assert) {
+        try {
+            createMenu({
+                items: [{ text: 'Item text', url: 'http://some_url', linkAttr: 'string' }]
+            });
+        } catch(e) {
+            assert.ok(false, `error ${e.message}`);
+        } finally {
+            assert.ok(true, 'no error');
+        }
+    });
```

Comment:
How should this api work if `item.template` is specified?
Let's add a test for this and add info in docs

Reply:
If item.template is specified, we can't control what text will be displayed. User need apply link in the template by himself.
I added this limitation in the specification for further describing in the doc. And test will be added

Reply:
Don't forget to add this to a docs card